.rubocop.yml

```diff
@@ -0,0 +1,6 @@
+StringLiterals:
+  Enabled: false
```

Comment:
You're missing the `Style/` prefix in the preceding line and instead of simply disabling the check, it's better to reconfigure it to our preferred style:

```yaml
Style/StringLiterals:
  EnforcedStyle: double_quotes
```

You also most likely want to add the following for consistency with the above and with Homebrew code in general:

```yaml
Style/StringLiteralsInInterpolation:
  EnforcedStyle: double_quotes
```